lib/membrane_rtc_engine/endpoints/webrtc/rtp_munger.ex

```diff
@@ -80,33 +83,57 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.RTPMunger do
       rtp_munger
       | highest_incoming_seq_num: buffer.metadata.rtp.sequence_number - 1,
         seq_num_offset: seq_num_offset,
-        timestamp_offset: timestamp_offset
+        timestamp_offset: timestamp_offset,
+        cache: Cache.new()
     }
   end
 
-  @spec munge(t(), Membrane.Buffer.t()) :: {t(), Membrane.Buffer.t()}
+  @spec munge(t(), Membrane.Buffer.t()) :: {t(), Membrane.Buffer.t() | nil}
+  def munge(rtp_munger, %Membrane.Buffer{} = buffer)
```

Comment:
As discussed, I don't think these paddings are a problem. It would be beneficial to drop them, as it would drop bitrate requirements for the track, but I don't it is problem quite yet

Reply:
Yeap, we can do this later in another PR but I would change API by adding `generate_padding_packet` function